examples/nat_scapy_tests.py

```diff
@@ -144,9 +144,11 @@ def run_server2(serverport=12022, natport=35002):
     ackn += 1
     print "Replying with Ack"
     send(ip/TCP(sport=serverport,dport=natport,flags="A", seq=seqn,ack=ackn))
+    time.sleep(1)
 
     ## Should now be unable to use the connection
     # Send something
+    time.sleep(1)
     seqn += 1
     ackn += 1
```

Comment:
So is the assumption that both `seqn` and `ackn` are always incremented 
by one? Does this mean that all segments contain a single byte?

On 2016-07-14 11:17, TMVector wrote:
> In examples/nat_scapy_tests.py [1]:
> 
>> 
>> ## Should now be unable to use the connection
>> # Send something
>> + time.sleep(1)
>> seqn += 1
>> ackn += 1
> 
> It's a feeble attempt to imitate the correct TCP protocol. As I said
> in another comment, these tests aren't very robust.
> 
> --
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub [2], or mute the
> thread [3].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/niksu/pax/pull/9#discussion_r70781857
> [2]
> https://github.com/niksu/pax/pull/9/files/6f58250655695a856c9c7e6c34ca2bb0edb2566b..d324c1fe3f83ce25053e4bc3fbfe0f249889a8de#r70781857
> [3]
> https://github.com/notifications/unsubscribe/ABDTTu3_fPLh4hnHS2AmSDxxiY0MQ-_hks5qVgzHgaJpZM4JLXUX

-- 
http://www.cl.cam.ac.uk/~ns441/